playground/lib/home/home_screen.dart

```diff
@@ -125,6 +145,25 @@ class _HomeScreenState extends State<HomeScreen> {
                           ),
                         ),
                         const Text('RTL'),
+                        Expanded(
+                            child: Center(
+                          child: IconButton(
+                            key: _attachedRandomKey,
+                            icon: const Icon(Icons.anchor),
+                            onPressed: () {
+                              WoltModalSheet.show(
+                                context: context,
+                                modalTypeBuilder: (_) =>
+                                    AttachedFloatingBottomSheetType(
+                                  anchorKey: _attachedRandomKey,
+                                  alignment: Alignment.bottomCenter,
+                                ),
+                                pageListBuilder: (_) =>
+                                    [AdjustTimeNotificationPage()],
```

Comment:
Can you add an option to hide the back button in this page? Maybe adding a boolean field `isFirstPage` and setting it true by default, but setting it false in this case, then hide the back button accordingly?

![image](https://github.com/user-attachments/assets/a62137f2-e5ca-4663-946a-e91648ffac01)

Reply:
Exchanged this for the modal used elsewhere in the playground